ui/revenuecatui/build.gradle

```diff
@@ -86,6 +86,7 @@ dependencies {
     testImplementation libs.coroutines.test
     testImplementation libs.kotlinx.serialization.json
     testImplementation libs.androidx.test.compose
+    testImplementation libs.androidx.test.compose.manifest
```

Comment:
Tests [failed on CI due to a manifest merger issue](https://app.circleci.com/pipelines/github/RevenueCat/purchases-android/12328/workflows/1ce073b2-3c74-4e05-bac4-6a1f855f6cbe/jobs/53596/tests).

This was already added as `debugImplementation`, but we run tests for the release variant (as well), e.g. `testDefaultsReleaseUnitTest`, so this makes sure we have it in any test variant, whether debug or release.